api_types.ts

```diff
@@ -515,6 +515,16 @@ export interface FetchRequest {
    * wish to make an unauthenticated supporting request as part of a formula implementation.
    */
   disableAuthentication?: boolean;
+  /**
+   * By default, any fetch response that more than 4MB will be rejected with
```

Comment:
The main goal is to prevent OOMs. David's marshaling change may fix the image transfer OOM issue but we may still explode with JIMP anyway. It would be smart for the G-Drive pack to simply skip thumbnail files larger than e.g. 100kb.

And it's better to do cancel the request before the pack receives the full image.

Reply:
`simplicity vs flexibility` -> not sure if we have a clear direction on this yet but I agree with your understanding for differences between citizen vs. sophisticated developers. I can see multiple ways how the experience may develop differently in the future (e.g. citizen developers go use a UI-based no-code pack builder). Can't say where we are heading to eventually.